author.php

```diff
@@ -26,16 +28,34 @@
 		<div class="columns padding-vertical-larger">
 			<div class="column is-8">
 				<?php
-				if ( have_posts() ) :
-					while ( have_posts() ) :
+				if (have_posts()) :
+					echo "<h2 class='mb-2'>" . join(" ", get_user_meta(get_queried_object()->ID)["first_name"]) . "'s Posts</h2>";
```

Comment:
- let's use the context variable for improved readability
- Why do we use `join` here?

```suggestion
					echo "<h2 class='mb-2'>" . join(" ", $author_first_name) . "'s Posts</h2>";
```